### ISSUE TITLE
Add test support for multiple scribe instances

### ISSUE THREAD

**Issue by cutandpastey** [guardian/scribe-plugin-noting#82]
Where we have functionality which involves multiple scribe instances i.e. collapse/expand all notes we should add tests.  

Currently this is difficult to do because the scribe selector (`.scribe`) is hard coded in the test harness, unless i'm totally missing something which is possible.  If we can get this fixed (see: https://github.com/guardian/scribe-test-harness/issues/13) we could add these tests into the test suite.

**Comment by hmgibson23** [comment]
Yes this is right. It's almost impossible currently to test against multiple scribe instances. We tried to do this once for  https://github.com/guardian/scribe/pull/262 but it proved pretty tricky.

**Comment by cutandpastey** [comment]
Yeah I also tried this with the placement of scribe markers and failed miserably. Perhaps the solution is to be able to pass the selector to `initializeScribe()` so you could call it twice, OR add `window.Scribe` so you could instantiate scribe again on a new element?
